Riot/Modules/Room/Views/BubbleCells/Call/RoomBaseCallBubbleCell.swift

```diff
@@ -0,0 +1,113 @@
+// 
+// Copyright 2020 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import Reusable
+
+class RoomBaseCallBubbleCell: MXKRoomBubbleTableViewCell {
```

Comment:
Should inherit `BaseBubbleCell` seems that we missed the inheritance on key verification cells :/

Reply:
I've decided not to use `BaseBubbleCell` here, as it has some overhead like avatar image, username etc, which we won't use in the call tiles.